frontend/src/metabase/entities/containers/EntityObjectLoader.jsx

```diff
@@ -52,18 +52,21 @@ const CONSUMED_PROPS: string[] = [
 ];
 
 @entityType()
-@connect((state, { entityDef, entityId, ...props }) => {
-  if (typeof entityId === "function") {
-    entityId = entityId(state, props);
-  }
-  return {
-    entityId,
-    object: entityDef.selectors.getObject(state, { entityId }),
-    fetched: entityDef.selectors.getFetched(state, { entityId }),
-    loading: entityDef.selectors.getLoading(state, { entityId }),
-    error: entityDef.selectors.getError(state, { entityId }),
-  };
-})
+@connect(
+  (state, { entityDef, entityId, selectorName = "getObject", ...props }) => {
```

Comment:
Can you add `selectorName` to the `Props` type as well as `CONSUMED_PROPS`? The latter is in case you stack multiple of the `Entity*Loader` HoCs, you don't want `selectorName` from the first to be passed to subsequent ones.

Also go ahead and add `selectorName` to `EntityListLoader`.